### PR TITLE
Publish bundles to v1.0 and fix doc consistency between beta and v1.0

### PR DESCRIPTION
**Pull request by kevklam**

### CODE REVIEW

File: api-reference/v1.0/api/bundle-additem.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+```http
+POST https://graph.microsoft.com/beta/drive/bundles/{bundle-id}/children
+Content-Type: application/json
+
+{
+  "id": "123456!87"
+}
+```
+# [C#](#tab/csharp)
```
Comment (Lauragra) [inline api-reference/v1.0/api/bundle-additem.md:69]: Global: When you move content to v1.0, you need to remove the snippet links from the files, including the HTTP H1 on line 58 in this case, and the characters in line 85.

**Comment by kevklam** [comment]
@Lauragra The ```http block stays in though right? just the H1's and links removed?

**Comment by kevklam** [comment]
@Lauragra Is this OK or do I need to delete the snippet files as well?

**Comment by Lauragra** [comment]
> @Lauragra Is this OK or do I need to delete the snippet files as well?

Yes, sorry, remove the actual snippet files from the PR as well. Thanks.

**Comment by kevklam** [comment]
@Lauragra OK, hopefully this is ready to go now.

File: api-reference/v1.0/api/bundle-additem.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+## Request body
+
+The request body includes the identifier for an item that should be added to the bundle's children collection.
+
+## Response
+
+If successful, the response is `204 No Content`.
+
+Read the [Error Responses][error-response] topic for more information about how errors are returned.
```
Comment (Lauragra) [inline api-reference/v1.0/api/bundle-additem.md:49]: Global: Please revise this sentence to "For information about error responses, see [Error responses]."

**Comment by Lauragra** [comment]
@kevklam , the APIs aren't live currently in the v1.0 metadata so I'm removing Ready to merge. Please reset when the APIs are released.

**Review by learafa — APPROVED** [review]

**Comment by kevklam** [comment]
@Lauragra I just noticed the localizationpriority apidoctor errors and fixed them. Not sure if there was anything else blocking this PR when you removed the 'ready to merge' flag before, please let me know

**Comment by isvargasmsft** [comment]
Hello @kevklam, 

What's the status of this PR? Is the schema already available for external customers to use?

**Comment by kevklam** [comment]
Yes. Please merge it. I forgot about this pr as it's been months since I last fixed conflicts and pushed.

Thanks,
Kevin

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by isvargasmsft** [comment]
@kevklam would you be so kind to resolve the conflicts in the PR, please?

File: api-reference/v1.0/api/bundle-list.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+GET /drive/bundles
+```
+
+## Optional query parameters
+
+This method supports the [OData Query Parameters][] to filter and shape the response.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-list.md:36]: For consistency, please use the boilerplate text here:

This method supports the \`$filter\` [OData query parameters](/graph/query-parameters) to help customize the response.

File: api-reference/v1.0/api/bundle-list.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+You can't use the `expand=children` query parameter when enumerating bundles.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-list.md:38]: Please, replace "when enumerating" with "to list".

File: api-reference/v1.0/api/bundle-list.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+|:------------- |:------------ |
+| Authorization | Bearer \{token\}. Required. |
+
+## Request body
+
+Do not supply a request body with this method.
+
+## Response
+
+If successful, this request returns the list of bundle items defined for the drive.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-list.md:52]: For consistency, please use the boilerplate text here:

- If successful, this method returns a \`200 OK\` response code and a collection of bundle[reference link] objects in the response body.

OR

- If successful, this method returns a \`200 OK\` response code and a collection of bundle[reference link] items defined for the drive[reference link] in the response body.

File: api-reference/v1.0/api/bundle-list.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+  "@odata.nextLink": "https://..."
+}
+```
+
+The response object shown here might be shortened for readability.
+
+
+### Example 2: List all photo albums in a drive
+
+To filter the list of bundles returned from a request to the bundles collection, you can use the `filter` query string parameter to specify the type of bundle to return by checking for the existence of a facet on the bundle:
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-list.md:105]: Please, replace the colon with a period.

File: api-reference/v1.0/api/bundle-removeitem.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+|:------------- |:------------ |
+| Authorization | Bearer \{token\}. Required. |
+
+## Request body
+
+Do not supply a request body with this method.
+
+## Response
+
+If successful, the response is `204 No Content`.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-removeitem.md:44]: For consistency, please use the boilerplate text here:

- If successful, this method returns a \`204 No Content\` response code.

File: api-reference/v1.0/api/bundle-update.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# Update bundle
+
+Namespace: microsoft.graph
+
+Update the metadata for a [bundle][] of [driveItems][driveItem] by ID.
+You can only update the following metadata:
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-update.md:15]: For clarity and consistency, please move this information to the Request body section.

File: api-reference/v1.0/api/bundle-update.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+## Request headers
+
+| Name          | Description  |
+|:------------- |:------------ |
+| Authorization | Bearer \{token\}. Required. |
+| if-match      | eTag. Optional. If this request header is included and the eTag provided does not match the current eTag on the buncle, a `412 Precondition Failed` response is returned.
+
+## Request body
+
+In the request body, supply the values for relevant fields that should be
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-update.md:49]: For consistency, please use the boilerplate text here followed by the table of supported properties (Bundle name and Album `coverImageItemId`):

```
## Request body

[!INCLUDE [table-intro](../../includes/update-property-table-intro.md)]

|Property|Type|Description|
|:---|:---|:---|
```

For an example, see https://raw.githubusercontent.com/microsoftgraph/microsoft-graph-docs/c6cd70308baebd22ff36854ea46432be2630fa94/api-reference/beta/api/cloudpcorganizationsettings-update.md.

File: api-reference/v1.0/api/bundle-update.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+updated. Existing properties that are not included in the request body
+will maintain their previous values or be recalculated based on changes to other
+property values. For best performance, don't include existing values that haven't changed.
+
+## Response
+
+If successful, this method returns a [driveItem][] resource that represents the updated bundle in the response body.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/bundle-update.md:56]: For consistency, please use the boilerplate text here:

- If successful, this method returns a \`200 OK\` response code and a [driveItem][] object in the response body.

OR

- If successful, this method returns a \`200 OK\` response code and a [driveItem][] object that represents the updated bundle in the response body.

File: api-reference/v1.0/resources/album.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+## Properties
+
+| Property name     | Type   | Description
+|:------------------|:-------|:------------------------------------------------
+| coverImageItemId | String | Unique identifier of the [driveItem][] that is the cover of the album.
+
+**Note:** If a **coverImageItemId** has not been set before, the thumbnails for an album are chosen automatically.
+After **coverImageItemId** has been set, the thumbnails for an album will always be the item associated with that id.
+You can override the default cover by PATCHing the [bundle item][bundle] and setting the **coverImageItemId** property on the `album` to the id of an image contained within the album.
+To remove a custom-set cover, you can set the **coverImageItemId** property to null, and a default one will be chosen automatically again.
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/album.md:24]: For consistency, please add backticks to null.

File: api-reference/v1.0/resources/album.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+A photo album is a way to virtually group [driveItems][driveItem] with [photo][] facets together in a [bundle][]. Bundles of this type will have the **album** property set on the [bundle][] resource.
+
+## Properties
+
+| Property name     | Type   | Description
+|:------------------|:-------|:------------------------------------------------
+| coverImageItemId | String | Unique identifier of the [driveItem][] that is the cover of the album.
+
+**Note:** If a **coverImageItemId** has not been set before, the thumbnails for an album are chosen automatically.
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/album.md:21]: For better visibility, please replace highlight this note using the following format:

```
>**Note:** If a **coverImageItemId** has not been set before, the thumbnails for an album are chosen automatically. After **coverImageItemId** has been set, the thumbnails for an album will always be the item associated with that ID. You can override the default cover by PATCHing the [bundle item][bundle] and setting the **coverImageItemId** property on the `album` to the ID of an image contained within the album. To remove a custom-set cover, you can set the **coverImageItemId** property to `null`, and a default one will be chosen automatically again.
```

File: api-reference/v1.0/resources/album.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+After **coverImageItemId** has been set, the thumbnails for an album will always be the item associated with that id.
+You can override the default cover by PATCHing the [bundle item][bundle] and setting the **coverImageItemId** property on the `album` to the id of an image contained within the album.
+To remove a custom-set cover, you can set the **coverImageItemId** property to null, and a default one will be chosen automatically again.
+
+## JSON representation
+
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/album.md:27]: For consistency, please add the following boilerplate text here:

- The following is a JSON representation of the resource.

Please, apply this change also here: 

- api-reference/v1.0/resources/bundle.md

File: api-reference/v1.0/resources/bundle.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+
+The `bundle` facet on a [driveItem][] identifies an item as a bundle and groups bundle-specific information into a single structure. It is only included on [driveItem][] resources returned from the **bundles** endpoint.
+
+Note that the `bundle` resource type itself is not an entity of its own, and is only a facet on a [driveItem][]. The `bundles` collection on a [drive][] is of type [driveItem][], not `bundle`.
+
+## Methods
+
+|                        Method             |         Return type      | Description        |
+| :---------------------------------------- | :----------------------- | :------------------|
+| [List bundles][bundle-list]               | [driveItem][] collection | List all bundles in a drive |
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/bundle.md:23]: Please, add a period at the end of the sentence. Please, apply this change to all descriptions in this table.

File: api-reference/v1.0/resources/bundle.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+description: Facet describing a driveItem that is a logical grouping of other driveItems
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: resourcePageType
+---
+# bundle resource type
+
+Namespace: microsoft.graph
+
+A bundle is a logical grouping of files used to share multiple files at once. It is represented by a [driveItem][] entity containing a `bundle` facet and can be shared in the same way as any other driveItem.
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/bundle.md:13]: 1. Please, avoid using gerunds where possible; in this case, you can change "containing" to "that contains".

1. For consistency, please replace all appearances of \`bundle\` (lines 13, 15, and 17) with \*\*bundle\*\*.

**Comment by kevklam** [comment]
@JarbasHorst I have been wrestling with this PR trying to get it merged since Jun 17 2021, and the functionality has been live in production for the entire time. Note that this is a direct port of the beta docs, which have been published for years now. I have in good faith tried to deal with merge conflicts and other issues both technical and content feedback related, but every time I push a new iteration the PR just ends up sitting here until something new comes up and either more merge conflicts come up or someone finds another missing period or nitpicks on some other word to bold. My team and I cannot invest any further resources into this PR. I see only two possible choices at this point:
a) Close the PR, and leave the v1.0 API undocumented forever. I'm personally happy just pointing people at the beta (or internal) documentation and telling them "it works the same in v1.0, we just weren't allowed to publish the docs to v1.0."
b) Merge the PR as-is

**Comment by JarbasHorst** [comment]
Hello @kevklam,

I haven't been part of the previous content review work for this PR but I can understand your point. We will discuss this topic internally and get back to you.

Best wishes,
Jarbas

**Pull request merged**